lib/hyacinth/adapters/external_identifier_adapter/datacite/rest_api/v2/api.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+require 'json'
+class Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::RestApi::V2::Api
+  MINIMUM_REQUIRED_ATTRIBUTES_FOR_FINDABLE_DOI = [:title,
+                                                  :creators,
+                                                  :doi_url,
+                                                  :publisher,
+                                                  :publication_year,
+                                                  :resource_type,
+                                                  :doi_prefix].freeze
```

Comment:
This is also a case of a constant that's defined identically in another file, and neither one appear to be used anywhere.

Reply:
MINIMUM_REQUIRED_ATTRIBUTES_FOR_FINDABLE_DOI removed from code.